client/static/FAQ.md

```diff
@@ -4,21 +4,21 @@ This is a polkadot-testnet faucet. You can use this to load testnet tokens into
 
 ## What is a faucet?
 
-A faucet is a developer tool to get a small amount of testnet tokens (ROC) in order to test and troubleshoot a decentralized application or protocol before going live on Polkadot, where one must use real DOT.
+A faucet is a developer tool to get a small amount of testnet tokens (<NETWORK-TOKEN>) in order to test and troubleshoot a decentralized application or protocol before going live on Polkadot, where one must use real DOT.
```

Comment:
In the rendered version, it disappears.

<img width="196" alt="Screenshot 2023-06-05 at 10 26 26" src="https://github.com/paritytech/polkadot-testnet-faucet/assets/12039224/e5110f2a-0143-456d-9a34-c85dccfa7d85">

Reply:
How about simply skipping the symbol and talk about `tokens` only?

Reply:
That's weird. In both of my cases they appear:
![image](https://github.com/paritytech/polkadot-testnet-faucet/assets/8524599/bfe902f3-1483-47bc-873a-c8e45c5a241b)

![image](https://github.com/paritytech/polkadot-testnet-faucet/assets/8524599/e3bad6d5-ebd7-4d07-8551-0bbe743c2f08)

Could you try rebuilding the project and trying again? Maybe it was a cache problem?

Reply:
Ah nevermind me, I was simply looking at the rendered markdown in github UI - forgot for a moment that it is processed by the framework.

All is good. :)